From b2670989f2b74f0f33c83ba18912f8a131814a77 Mon Sep 17 00:00:00 2001
From: Sunita Nadampalli <nadampal@amazon.com>
Date: Mon, 27 Jun 2022 01:06:12 +0000
Subject: [PATCH] Add Arm Compute Library (ACL) build option for torch-xla

ACL build support has been added under the build flag: XLA_CPU_USE_ACL
---
 build_torch_xla_libs.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/build_torch_xla_libs.sh b/build_torch_xla_libs.sh
index 82b70559..d0313150 100755
--- a/build_torch_xla_libs.sh
+++ b/build_torch_xla_libs.sh
@@ -66,6 +66,10 @@ if [[ "$XLA_CUDA" == "1" ]]; then
   OPTS+=(--config=cuda)
 fi
 
+if [[ "$XLA_CPU_USE_ACL" == "1" ]]; then
+  OPTS+=("--define=build_with_acl=true")
+fi
+
 if [ "$CMD" == "clean" ]; then
   pushd $THIRD_PARTY_DIR/tensorflow
   bazel clean
-- 
2.25.1

